packages/forms/frontend/sirius-components-forms/src/propertysections/ButtonPropertySection.tsx

```diff
@@ -118,18 +119,16 @@ export const ButtonPropertySection: PropertySectionComponent<GQLButton> = ({
   const { addErrorMessage, addMessages } = useMultiToast();
 
   useEffect(() => {
-    if (!loading) {
-      if (error) {
-        addErrorMessage('An unexpected error has occurred, please refresh the page');
-      }
-      if (data) {
-        const { pushButton } = data;
-        if (isErrorPayload(pushButton) || isSuccessPayload(pushButton)) {
-          addMessages(pushButton.messages);
-        }
+    if (error) {
+      addErrorMessage(error.message);
```

Comment:
There are a lot of places where you have changed the error message logged. These messages have been setup this way for a reason.